oxide-auth/src/primitives/generator.rs

```diff
@@ -318,15 +318,15 @@ mod time_serde {
     use chrono::{TimeZone, Utc};
 
     use serde::ser::{Serializer};
-    use serde::de::{Deserialize, Deserializer};
+    use serde::de::{Deserialize, Deserializer, Error};
 
     pub fn serialize<S: Serializer>(time: &Time, serializer: S) -> Result<S::Ok, S::Error> {
         serializer.serialize_i64(time.timestamp())
     }
 
     pub fn deserialize<'de, D: Deserializer<'de>>(deserializer: D) -> Result<Time, D::Error> {
         let as_timestamp: i64 = <i64>::deserialize(deserializer)?;
-        Ok(Utc.timestamp(as_timestamp, 0))
+        Utc.timestamp_opt(as_timestamp, 0).single().ok_or(Error::custom("Invalid timestamp"))
```

Comment:
It would be more informative if this used `Error::invalid_value` instead of a barebones message.

Reply:
fixed ⏲️